core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/dao/JPAPlainSchemaDAO.java

```diff
@@ -101,6 +105,16 @@ public PlainSchema find(final String key) {
         return query.getResultList();
     }
 
+    @Override
+    public <T extends PlainAttr<?>> boolean hasAttrs(PlainSchema schema, Class<T> reference) {
+        String plainAttrTable = getPlainAttrTable(reference);
+        Query query = entityManager().createNativeQuery("SELECT COUNT(" + plainAttrTable + ".ID) FROM PLAINSCHEMA JOIN "
```

Comment:
You are still using the literal `"PLAINSCHEMA"` instead of `JPAPlainSchema.TABLE`